unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuilderInWorldAudioHandler.cs

```diff
@@ -69,9 +70,11 @@ private void AddListeners()
         entityHandler.OnDeleteSelectedEntities += OnAssetDelete;
         modeController.OnChangedEditModeState += OnChangedEditModeState;
         DCL.Environment.i.world.sceneBoundsChecker.OnEntityBoundsCheckerStatusChanged += OnEntityBoundsCheckerStatusChanged;
-
-        DCL.Tutorial.TutorialController.i.OnTutorialEnabled += OnTutorialEnabled;
-        DCL.Tutorial.TutorialController.i.OnTutorialDisabled += OnTutorialDisabled;
+        if (DCL.Tutorial.TutorialController.i != null)
+        {
+            DCL.Tutorial.TutorialController.i.OnTutorialEnabled += OnTutorialEnabled;
+            DCL.Tutorial.TutorialController.i.OnTutorialDisabled += OnTutorialDisabled;
+        }
```

Comment:
If we don't plan to add a tutorial in BiW in the short/medium term (I don't know what are the plans for it), maybe we should clean all code related to it.

Reply:
This part is a fix because both pieces of music could play at the same time. You could enter builder in world with the tutorial music , so this way we ensure that only 1 music is playing. However, I agree with remove all the code related to it!